tariff-logic/taric-footwear.test.js

```diff
@@ -15,674 +15,1000 @@ test.each([
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'yes' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'men' }
+    ],
+    '6403999610'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'men' }
+    ],
+    '6403999690'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'yes' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'women' }
+    ],
+    '6403999810'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'women' }
+    ],
+    '6403999890'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'yes' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'unisex/other' }
+    ],
+    '6403999310'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'unisex/other' }
+    ],
+    '6403999390'
+  ],
+  // 6404
+  ['640420', [], { code: '640420', questionKey: 'slippers' }],
+  ['640420', [{ questionKey: 'slippers', answerKey: 'yes' }], '6404201000'],
+  ['640420', [{ questionKey: 'slippers', answerKey: 'no' }], '6404209000'],
+  [
+    '640411',
+    [
+      { questionKey: 'sole', answerKey: 'plastic' },
+      { questionKey: 'sports', answerKey: 'yes' }
+    ],
+    '6404110000'
+  ],
+  [
+    '640419',
+    [
+      { questionKey: 'sole', answerKey: 'plastic' },
```

Comment:
We have a lot of irrelevant questionAnswers here. We also lack the scenario where we get the slippers question back

Reply:
Irrelevant questionAnswers removed and order changed to numerical for 6404